pages/portfolio.html

```diff
@@ -4,9 +4,59 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>My Work</title>
+  <link href="/stylesheets/style.css" rel="stylesheet">
+  <link href="/stylesheets/portfolio-style.css" rel="stylesheet">
 </head>
+
 <body>
+  <main class="container">
+    <header><h1>tiffini hyatt</h1></header>
+
+    <nav>
+      <menu class="nav-links">
+        <li id="home"><a href="pages/index.html"><h4>home</h4></a></li>
+        <li id="portfolio"><a href="pages/portfolio.html"><h4>work</h4></a></li>
+        <li id="about"><a href="pages/about.html"><h4>about me</h4></a></li>
```

Comment:
We're jumping from an `h1` for the title, down to an `h4` in the nav links across all the pages. In the MDN docs it's highly suggested to not skip headings:
> A common navigation technique for users of screen reading software is jumping from heading to quickly determine the content of the page. Because of this, it is important to not skip one or more heading levels. Doing so may create confusion, as the person navigating this way may be left wondering where the missing heading is.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#navigation

If these are truly content headings, the suggestion would be to use `h2` and style it as needed to match the smaller font size desired. If the `h4` tags were being used for the default styling then we should remove them and either style the `li` elements or wrap the text in `p` elements and style those.